src/screens/Document/reducers.js

```diff
@@ -1,16 +1,33 @@
 import { SUBMIT_DOCUMENT, RESET_DOCUMENT_SAVED } from '../../store/actionTypes';
 
-const documentReducer = (state = { documentWasSaved: false }, action) => {
+const initialState = {
+  documentWasSaved: false,
+  hasDocumentBeenEdited: false
```

Comment:
I don't think you actually ended up using hasDocumentBeenEdited anywhere (it's passed down as props to Home/index.js, but then it's not used in the logic). Pls remove this property from the redux state if that's the case.